main.js

```diff
@@ -324,6 +355,103 @@ define(function (require, exports, module) {
         toggleCmd.setChecked(visible);
     }
 
+    // Support custom theme
+    function _onSelectCSS(err, fi) {
+        if (err) {
+            console.error(err);
+            Dialogs.showModalDialog(
+                "",
+                "Markdown Preview",
+                "There was an error with your selection."
+            );
+            return;
+        }
+
+        if (fi.length === 1) {
+            _prefs.set("theme", "custom");
```

Comment:
Once a custom theme is set, there is no way to change it to a different theme (short of renaming the file to force a reset). There should be a way to switch to a different CSS file. 

Here is one possible suggestion: keep the Custom menu item, but add a new menu item whenever a custom CSS file is selected. This way the theme menu is populated with all of the built-in themes, and all themes selected by the user. If you do this, the Custom menu item should have an ellipses at the end.